XmlFormatter/src/Settings/Provider/DataStructure/SerializableSettingContainer.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.Collections.Generic;
+
+namespace XmlFormatter.src.Settings.Provider.DataStructure
+{
+    /// <summary>
+    /// A saveable settings container
+    /// </summary>
+    public class SerializableSettingContainer
+    {
+
+        /// <summary>
+        /// Public access to all the scopes in this container
+        /// </summary>
+        public List<SerializableScope> Scopes;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make this field 'private' and encapsulate it in a 'public' property.](https://app.codacy.com/manual/simonaberle/XmlFormatter/pullRequest?prid=5053335)